CMakeLists.txt

```diff
@@ -39,33 +39,16 @@ set(CPACK_DEBIAN_PACKAGE_DESCRIPTION ${CPACK_PACKAGE_DESCRIPTION})
 set(CPACK_DEBIAN_PACKAGE_SECTION "games")
 include(CPack)
 
+set(CHIAKI_IS_SWITCH ${NSWITCH})
```

Comment:
the NSWICTH flag comes from /opt/devkitpro/switch.cmake
Hope they will not change it in a future release.
maybe duplicate the flag in our local cmake/switch.cmake toolchain might help.